### PR TITLE
Feature: profile picture in navbar for logged in users

### PR DESCRIPTION
**Pull request by neosb**
For users that logs in, for their convenience, there is now a Facebook profile picture in navbar next to their Facebook name. Avatar is 20 pixel high and 20 pixels wide, thus rather small, but it doesn't stand out beetween list items. I have added a css class to `ul` making the list look horizontally aligned. This changes apply to `_navbar.haml`.

I have also changed `layout.scss` stylesheet to display text better using 15 pixel font size for part on the left and 500 font-weight for all elements of navbar, so that they are more readable.

There are also changes added bootstrap classes to `_event.haml` partial and `events#show` action.

### CODE REVIEW

File: app/views/application/_navbar.haml [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
           %p.navbar-text
+            = image_tag current_user.image, size: '20x20', alt: "Profile image"
             = current_user.name
-            
-            = link_to logout_path do
-              %i.fa.fa-sign-out
-              Sign out
+        %li
+          = link_to logout_path do
+            %i.fa.fa-sign-out
```
Comment (cintrzyk) [inline app/views/application/_navbar.haml]: do we have some helper method for creating fontawesome icon provided by gem? like `fa_icon('sign-out')`? if not please create this kind of helper and use it in navbar :)

File: app/views/application/_navbar.haml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   .container
     .navbar-header
       = link_to "Meecon", root_path, class: 'navbar-brand'
-    %ul.nav.navbar-nav.navbar-right
+    %ul.nav.navbar-nav.navbar-right.navbar-font-size
```
Comment (cintrzyk) [inline app/views/application/_navbar.haml]: nice one! but consider case when you would like to add more styles to this `ul` element like `margin` and adding extra class `navbar-margin` wouldn't be a good idea. What I'm trying to say is if you could rename the css class to something related to this specific element for example `.navbar-navigation` adding more styles will be very handy :)

**Comment by cintrzyk** [comment]
good job, looks much nicer ;)

File: app/views/application/_navbar.haml (hunk 1 above)
Comment (neosb) [inline app/views/application/_navbar.haml]: In gem `font-awesome-sass` there is a helper method `icon 'plus', 'Add something', class: 'my-class', id: 'my-id'` which will output icon with text. You could omit the `'Add something'` string to output just icon in your view.

Comment (cintrzyk) [inline app/views/application/_navbar.haml]: great! lets use it :)

**Comment by neosb** [comment]
Fixed code according to @cintrzyk comments. I have also changed `events/show.haml` view to use `icon` helper method from `font-awesome-sass` gem.

**Comment by cintrzyk** [comment]
looks :+1:

**Pull request merged**